aip-console-tools-cli/src/main/java/com/castsoftware/aip/console/tools/commands/DeliverVersionCommand.java

```diff
@@ -95,6 +95,9 @@ public class DeliverVersionCommand implements Callable<Integer> {
                     + " if specified without parameter: ${FALLBACK-VALUE}",
             fallbackValue = "true")
     private boolean autoCreate = false;
+    
+    @CommandLine.Option(names = {"-css", "--css-server"}, description = "CSS Server name that will host the application data: format is host:port ")
```

Comment:
Its using the Server Name and not `host:port` specifically right ? if I check in console, I see one of my css connection like `localhost:2284/postgres`. Maybe add that as an example ?

```suggestion
     @CommandLine.Option(names = {"-css", "--css-server"}, description = "CSS Server name that will host the application data. Format will be host:port/databaseName and can be checked on AIP Console's Global Configuration page.")
```